src/nvim/drawline.c

```diff
@@ -411,7 +411,7 @@ static void get_statuscol_str(win_T *wp, linenr_T lnum, int row, int startrow, i
   bool use_cul = use_cursor_line_sign(wp, lnum);
 
   // Set num, fold and sign text and attrs, empty when wrapped
-  if (row == startrow) {
+  if (!wrapped) {
```

Comment:
This is not the same, and incorrect. Please undo, `wrapped` also checks for `filler_lines`, perhaps the variable name is inconvenient. I'll see if I can add a test that would have failed here.

I think the `get_vim_var_nr(VV_WRAP)` changes in `build_stl_str_hl()` are sufficient for this PR?

Reply:
Yeah, it was a stale change from when I was experimenting with the code